apps/rpc/src/invariant.ts

```diff
@@ -13,7 +13,7 @@ const logger = getLogger("db-query-invariant-checker")
 export function parseOrReport<T>(schema: z.ZodSchema<T>, value: T): T {
   const result = schema.safeParse(value)
   if (!result.success) {
-    logger.error("Database failed to parse value into schema: %s emitted for object %o", result.error.message, value)
+    logger.error("Database failed to parse value into schema: %s emitted for object %o", result, value)
```

Comment:
Needs %o for the formatter to format it nicely
```suggestion
    logger.error("Database failed to parse value into schema: %o emitted for object %o", result, value)
```

Reply:
Nice thanks